docs/.vuepress/config.js

```diff
@@ -57,14 +57,14 @@ module.exports = {
       binary: 'evmosd',
       testnet_denom: 'Photon',
       testnet_ticker: 'PHOTON',
-      rpc_url: 'http://localhost:8545/',
-      rpc_url_testnet: 'http://arsiamons.rpc.evmos.org:8545',
+      rpc_url: 'https://rpc.evmos.org:8545',
+      rpc_url_testnet: 'https://olympusmons.rpc.evmos.org:8545',
```

Comment:
We'll have to get rid of the port in this case. It'll either have to be `http://olympusmons.rpc.evmos.org:8545` or `https://olympusmons.rpc.evmos.org`